mage_ai/data_preparation/templates/transformers/transformer_actions/column/one_hot_encode.py

```diff
@@ -0,0 +1,18 @@
+{% extends "transformers/transformer_actions/action.jinja" %}
+{% block imports %}
+{{ super() -}}
+{% endblock %}
+{% block action %}
+    """
+    Execute Transformer Action: ActionType.ONE_HOT_ENCODE
+
+    Docs: https://github.com/mage-ai/mage-ai/blob/master/docs/actions/transformer_actions/README.md#one-hot-encode
```

Comment:
Could you add doc for this transformer action in this doc: https://github.com/mage-ai/mage-ai/blob/master/docs/guides/transformer-blocks.mdx#transformer-action-reference? And you can use doc link https://docs.mage.ai/guides/transformer-blocks#one-hot-encode here.